kibble/api/prices.go

```diff
@@ -126,11 +126,31 @@ func processPrices(details prices, site *models.Site, itemIndex models.ItemIndex
 		}
 	}
 
+	// loop over the retrieved list of films that belong to plans
+	// assign a slice of maps to each film e.g. [{"plan/123" : prices }, ...]
+	for _, filmPlans := range details.Plans {
+		if film, ok := site.Films.FindFilmBySlug(filmPlans.Item); ok {
+
+			filmPlanPrices := make(map[string]models.PriceCollection, 0)
+
+			for _, filmPlan := range filmPlans.Plans {
+				if plan, err := site.Plans.FindPlanBySlug(filmPlan); err == nil {
```

Comment:
Less concerned about the performance impact here but I may be underestimating it; if you think this can be done more efficiently be my guest.

Reply:
I attempted and don't think it can be. Any ideas?

Reply:
- FindFilmBySlug changed from for loop to object key selection in this PR https://github.com/shift72/kibble/pull/108
- resulted in 'significant improvement with more films'.
- Could add this to plans also but negligible (but definately later on, with  tv, bundles while we're at it, for consistency at least.)